### PR TITLE
Add ValidatorCollector

### PR DESCRIPTION
**Pull request by xepozz**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌

### CODE REVIEW

File: src/Collector/ValidatorCollector.php [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        'validator.count_invalid' => 'int',
+    ])]
+    public function getIndexData(): array
+    {
+        $count = count($this->validations);
+        $countValid = count(array_filter($this->validations, fn (array $data) => $data['result']));
+        $countInvalid = $count - $countValid;
+
+        return [
+            'validator.count' => $count,
```
Comment (rustamwin) [inline src/Collector/ValidatorCollector.php:53]: Let's follow naming convention (camelCase) :slightly_smiling_face:

Comment (xepozz) [reply src/Collector/ValidatorCollector.php:53]: Actually I want to change it to make more clear understanding of the origin ;)

Comment (xepozz) [reply src/Collector/ValidatorCollector.php:53]: Changed names in https://github.com/yiisoft/yii-debug/pull/133

**Review by rustamwin — APPROVED** [review]

**Comment by rustamwin** [comment]
Seems to need to adapt to the latest yiisoft/validator#255 changes.

**Review by samdark — APPROVED** [review]

**Pull request merged**